packages/ui-components/src/Button/README.md

```diff
@@ -0,0 +1,85 @@
+[back to components](../README.md)
+
+# Button
+
+A Button is a styled basic element to handle click-based user interactions.
+
+```javascript
+import React from "react";
+import { Button } from "@cockroachlabs/ui-components";
+
+export const ButtonExample = () => (
+  <Button>Default button</Button>
+  <Button type="primary-success">Success button</Button>
+  <Button type="primary-danger">Danger button</Button>
+  <Button type="secondary">Secondary button</Button>
+  <Button type="tertiary">Tertiary button</Button>
+  <Buttont size="small">Small button</Button>
+);
+```
+
+## Properties
```

Comment:
Having this conversation on [another PR](https://github.com/cockroachdb/ui/pull/41#discussion_r434848133) with @nathanstilwell -- I don't think documenting all properties of a component in a README is sustainable. The docs addon should allow us to add this commentary in the [props table](https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/props-tables.md).

Reply:
Great.  I definitely agree.  Root-level READMEs are already vectors for stale information to creep in; collections of them will almost certainly get stale very, very rapidly without laser-focused CR on that particular aspect (which seems like a bad use of time).